connector/connect/client/jvm-internal/src/test/scala/org/apache/spark/sql/connect/client/SparkConnectClientSuite.scala

```diff
@@ -0,0 +1,218 @@
+/*
```

Comment:
This file has been mostly moved, but
* DummySparkConnectService was moved to it's own util file
* two tests were moved to SparkSessionDummyIntegrationSuite

and it makes github see it as a full diff.